fix(curriculum): external calenders wiki in code interview (#46191)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md

# --description--

> The Gregorian calendar is a solar calendar with 12 months of 28–31 days each. The year in both calendars consists of 365 days, with a leap day being added to February in the leap years except three centurial years every 400 years.
> 
> The Discordian calender is aligned with the Gregorian calendar and begins on January 1. Its week consists of five days, there are 73 weeks per year. Every fourth year in the Discordian calendar, an extra day is inserted between Chaos 59 and Chaos 60
> 
> note that in Discordian calender Chaos 1, 3188 YOLD is January 1, 2022 in Gregorian calender.
> 
> Convert a given date from the Gregorian calendar to the Discordian calendar.

Reviewer: ```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28–31 days each. The year in both calendars consists of 365 days, with a leap day added to February in the leap years except for three centurial years every 400 years.

The Discordian calendar is aligned with the Gregorian calendar and begins on January 1. Its week consists of five days, there are 73 weeks per year. Every fourth year in the Discordian calendar, an extra day is inserted between Chaos 59 and Chaos 60

Note that in the Discordian calendar Chaos 1, 3188 YOLD is January 1, 2022, in the Gregorian calendar.

Convert a given date from the Gregorian calendar to the Discordian calendar.
```
